modules/report/src/main/Report.scala

```diff
@@ -53,10 +53,11 @@ object Report {
 
   case class Inquiry(mod: User.ID, seenAt: DateTime)
 
-  case class WithUser(report: Report, user: User, isOnline: Boolean) {
+  case class WithUser(report: Report, user: User, isOnline: Boolean, accuracy: Option[Int], room: Option[Room]) {
 
     def urgency: Int =
-      (nowSeconds - report.createdAt.getSeconds).toInt +
+      (if (room has Room.Cheat) accuracy getOrElse 0
```

Comment:
And this is only when you're in the Cheat view. If you're in the All view it will be sorted by date along with the rest.